src/rust/ide/view/graph-editor/src/component/node.rs

```diff
@@ -380,6 +440,18 @@ impl NodeModel {
     pub fn visualization(&self) -> &visualization::Container {
         &self.visualization
     }
+
+    fn set_error(&self, error:&Option<Error>) {
+        // TODO: This extra text field should not be required after #1026 has been finished.
+        //Instead we should get the error content as normal node output that is visible in the
+        //visualisation.
```

Comment:
I think this comment shouldgo to the text field definition place. Moreover, I'm not sure this is true, as we want to display errors even with visualisations closed.